docs/configuration.md

```diff
@@ -0,0 +1,97 @@
+Configuration DSL
+=================
+
+The configuration DSL is made up of key-value pairs, called `attributes`, which are grouped into `namespaces` and `collections`.
+
+Namespaces can accessed with blocks, or with a fluent interface:
+
+```ruby
+aws do |a|
+  a.region = 'us-west-1'
+end
+
+## Is the same as
+aws.region = 'us-west-1'
+```
+
+Collections are named sets. Like namespaces, they can be accessed with blocks, or a fluent interface:
```

Comment:
This is kind of an odd question. https://www.google.com/search?q=define%20collection&rct=j

Do you have an alternative?